pennylane/qnode.py

```diff
@@ -29,6 +29,24 @@
 from pennylane.tape import QuantumTape, make_qscript
 
 
+def _convert_to_interface(res, interface):
+    """
+    Recursively convert res to the given interface.
+    """
+    interface = INTERFACE_MAP[interface]
+
+    if interface in ["Numpy", "autograd"]:

```

Comment:
do we want to include autograd here? The return type should probably be a qml.numpy.tensor in that case
```suggestion
    if interface == "Numpy":
```

Reply:
Updated in newest commit.